src/bin/bat/app.rs

```diff
@@ -226,17 +224,14 @@ impl App {
             style_components,
             syntax_mapping,
             pager: self.matches.value_of("pager"),
-            use_italic_text: match self.matches.value_of("italic-text") {
-                Some("always") => true,
-                _ => false,
-            },
+            use_italic_text: matches!(self.matches.value_of("italic-text"), Some("always")),
```

Comment:
This could be:
``` rust
self.matches.value_of("italic-text") == Some("always")
```
instead, right?

Reply:
Yes, good catch, thank you!
changed in ba4c5d03463b74